### PR TITLE
Update UserMenu

### PR DESCRIPTION
**Pull request by rsandb**
now uses chakra components and includes score. Score needs to be implemented, currently is just a placeholder

### CODE REVIEW

File: website/src/components/Header/UserMenu.tsx [hunk 1]
```diff
@@ -29,12 +37,20 @@ export function UserMenu() {
...
         href: "https://github.com/LAION-AI/Open-Assistant/issues/new/choose",
         desc: "Report a Bug",
         icon: FiAlertTriangle,
       },
+      {
+        name: "FAQ",
+        href: "/faq",
+        desc: "Report a Bug",
+        icon: FiHelpCircle,
+      },
```
Comment (AbdBarho) [inline website/src/components/Header/UserMenu.tsx:53]: do we have a FAQ page?

Comment (fozziethebeat) [reply website/src/components/Header/UserMenu.tsx:53]: We don't.  We can drop this entry for now.

**Comment by fozziethebeat** [comment]
Approve after addressing @AbdBarho's request for changes.

**Review by fozziethebeat — APPROVED** [review]
Confirm that the requested change is done! merging.

**Pull request merged**